03LessonThree/in-class-project/daily-planet.html

```diff
@@ -1,10 +1,68 @@
+        <h1>The Daily Planet</h1>
+        <nav>
+            <a href="#">Sports</a>
+            <a href="#">Local</a>
+            <a href="#">Arts & Entertainment</a>
+        </nav>
+    </header>
+    <main>
+        <section>
+            <img src="http://interactives.dallasnews.com/2016/texas-olympians/images/lede-2400.jpg" />
+            <article>
+                <h2>Texas Gold</h2>
+                <h3><em>More than 100 athletes from the Lone Star State will compete in Brazil</em></h3>
+                <h4>By: Jori Epstein</h4>
+                <p>Texas will be well-represented at the 2016 Olympics in Rio de Janeiro, as over 100 athletes with ties across the state compete in Brazil.</p>
+                <p>We’re tracking all athletes with Texas ties as they compete for their respective nations from Aug. 5-21 in Rio. Texas natives, current residents, anyone who went to high school or university in The Lone Star State? You can find them all
+                    here.</p>
+                <p>The schedule below will update as athletes finish events with results, related stories and TV times. The athletes automatically sort alphabetically, but you can also sort by upcoming events — so you can map out your calendar perfectly
+                    to cheer on Texans.</p>
+            </article>
+        </section>
+    </main>
+    <aside>
+        <a href="https://www.insightly.com/"><img src="https://aka-cdn.adtechus.com/images/412/Ad0St1Sz529Sq0V0Id24953244.png" />
+        </a>
+    </aside>
+    <footer>
+        <table>
+            <thead></thead>
+            <tbody>
+                <tr>
+                    <td>
+                        <a href="https://www.facebook.com/sharer/sharer.php?u=http://interactives.dallasnews.com/2016/texas-olympians/">
+                            <img src="https://www.facebookbrand.com/img/fb-art.jpg" />
+                        </a>
+                    </td>
+                    <td>
+                        <a href="https://twitter.com/intent/tweet?text=Keep%20up%20with%20all%20of%20the%20Texas%20olympians%20with%20our%202016%20Summer%20Olympics%20tracker%20%7C%20@SportsDayDFW&url=http://interactives.dallasnews.com/2016/texas-olympians/">
+                            <img src="https://cdn3.iconfinder.com/data/icons/social-icons-5/607/Twitterbird.png" />
+                        </a>
+                    </td>
+                    <td>
+                        <a href="mailto:test@test.com">
+                            <img src="http://www.freeiconspng.com/uploads/email-icon-23.png" />
+                        </a>
+                    </td>
+                    <td>
+                        <a href="#">
+                            <img src="http://3835642c2693476aa717-d4b78efce91b9730bcca725cf9bb0b37.r51.cf1.rackcdn.com/Instagram_App_Large_May2016_200.png" />
+                        </a>
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+    </footer>
```

Comment:
You are missing one of the project requirements - Add copyright and company name to footer. If you can add and recommit, I can bump your grade up 5 points.